etna/experimental/prediction_intervals/empirical.py

```diff
@@ -0,0 +1,91 @@
+from typing import Sequence
+
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.experimental.prediction_intervals import BasePredictionIntervals
+from etna.experimental.prediction_intervals.utils import residuals_matrices
+from etna.pipeline import BasePipeline
+
+
+class EmpiricalPredictionIntervals(BasePredictionIntervals):
+    """Estimate prediction intervals using values of historical residuals.
+
+    1. Compute matrix of residuals  :math:`r_{it} = |\hat y_{it} - y_{it}|` using k-fold backtest, where :math:`i` is fold index.
+
+    2. Estimate quantiles levels, that satisfy the provided coverage, for the corresponding residuals distributions.
+
+    3. Estimate quantiles for each timestamp using computed residuals and levels.
+
+    `Reference implementation <https://www.sktime.net/en/stable/api_reference/auto_generated/sktime.forecasting.conformal.ConformalIntervals.html>`_.
+    """
+
+    def __init__(self, pipeline: BasePipeline, coverage: float = 0.95, cutoff_borders: bool = True, stride: int = 1):
+        """Initialize instance of ``EmpiricalPredictionIntervals`` with given parameters.
+
+        Parameters
+        ----------
+        pipeline:
+            Base pipeline or ensemble for prediction intervals estimation.
+        coverage:
+             Interval coverage. In literature this value maybe referred as ``1 - alpha``.
+        cutoff_borders:
```

Comment:
I'm not very sure that current parameter name is clear enough, it isn't obvious what it responsible for by its name. May be smth like "include_prediction" suits better.

Reply:
Changed name